modules/storages/config/locales/en.yml

```diff
@@ -251,22 +278,31 @@ en:
         oidc_cant_refresh_token: There was an error while trying to check your access to the storage. Please check the server logs for further information.
```

Comment:
Proposal for line 275:

```diff
-oauth_token_missing: OpenProject cannot test the user level communication with Nextcloud as there's no token for the current user.
+oauth_token_missing: OpenProject cannot test the user level communication with Nextcloud as the current user did not yet link their Nextcloud account.
```

The idea is, that we formulate the message in a way that's understandable/actionable for the user performing the check.

I am a bit sad that we just call it "Login to Nextcloud" in the rest of the UI, because it's not a good hint to tell a user to "Login to Nextcloud" (they can do that by opening a new Browser tab). So I just made the term "link" up here.

Reply:
no big deal, I changed the text